portfolio/src/main/webapp/script.js

```diff
@@ -20,6 +20,12 @@ document.addEventListener('DOMContentLoaded', function() {
   instances_two = M.Materialbox.init(elems_two);
 });
 
+//Load Navigation Bar and Footer when loading page
+document.addEventListener('DOMContentLoaded', function() {
```

Comment:
You've now got two event listeners triggered on the same event (DOMContentLoaded).
This is not inherently a problem but it can make code less traceable if say you had these scattered all over the file, and there is no guaranteed order these will run in so if they had any dependencies you could run into trouble (I know these don't have dependencies but just for best practice).
I think maybe the better way to do this would be to put all the code in one even handler and call out to sub functions or just seperate the code with comments if it's small snippets like this.

Reply:
Thanks @lorikeet-s! I have combined the 2 event listeners to 1, and added comments in between. Let me know whether there is anything else I can change/improve